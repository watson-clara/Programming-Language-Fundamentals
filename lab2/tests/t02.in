debug off
stopwords on
length 10
load tracks.txt
preprocess
sequence hello
sequence remember
sequence live
sequence laugh
sequence love
sequence to
sequence hate
sequence pearl
sequence problems
sequence meaning
sequence of
sequence life
sequence amor
sequence corazon
sequence gusto
sequence grande
sequence triste
sequence amo
sequence la
sequence perla
sequence first
sequence last
sequence big
sequence small
sequence never
sequence always
sequence forever
sequence say
sequence go
sequence do
sequence be
sequence see
sequence I
sequence think
sequence speak
sequence learn
sequence feel
sequence lost
sequence goodbye
sequence stop
