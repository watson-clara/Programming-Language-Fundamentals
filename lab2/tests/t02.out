hello little girl i don't know what you are my
remember me down my heart is love you are we
live it up my heart is love you are we
laugh after all i don't know what you are my
love you are my heart is your eyes child god
to
hate me down my heart is love you are we
pearl black white christmas song is my heart attack cenobites
problems
meaning life is my heart attack cenobites
of
life is my heart attack cenobites
amor de la vida es una noche mas que te
corazon de la vida es una noche mas que te
gusto es la vida loca toca mi amor de ti
grande amor de la vida es una noche mas que
triste bachata cha brasilia 5 minutes more than you are
amo assim assado
la vida es una noche de mi amor se me
perla de la vida es una noche mas que te
first time is my heart attack cenobites
last night long way you are my heart is love
big up my heart is love you are we go
small town bethlehem
never be your love you are my heart is it
always my heart is love you are we go home
forever young love you are my heart is your eyes
say goodbye my heart is love you are we go
go home christmas song is my heart attack cenobites
do you are my heart is love me down home
be your love you are my heart is it up
see you are my heart is love me down home
i don't know what you are my heart is love
think you are my heart is love me down home
speak low down my heart is love you are we
learn fly away me down my heart is love you
feel it up my heart is love you are we
lost my heart is love you are we go home
goodbye my heart is love you are we go home
stop loving you are my heart is love me down
